src/feed/Feed.ts

```diff
@@ -443,10 +448,18 @@ class Feed {
                 noCreateDataSet: true,
             });
         } else {
-            this._stx.updateChartData(quotes, null, {
-                allowReplaceOHL: true,
-            });
-            this._stx.createDataSet();
+            // we will reload the chart instead of rendering new ticks
+            // in case if the duration between the previous tick and the current one is longer than a minute
+            // to avoid disturbing animation on slower devices
+            if (current_tick_timestamp - this._last_tick_timestamp < 60) {
+                this._stx.updateChartData(quotes, null, {
+                    allowReplaceOHL: true,
+                });
+                this._stx.createDataSet();
+            } else {
+                this._mainStore.chart.refreshChart();
```

Comment:
Will this be called multiple times when multiple ticks come at once?

Reply:
No, it will only be executed once when the difference between the previous and the current tick is above 60 seconds. After that the chart will refresh and reset the whole feed, so it will have at least another 60 seconds before the next execution.